uPortal-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -90,15 +90,15 @@
         <filter-class>org.jasig.cas.client.validation.Cas20ProxyReceivingTicketValidationFilter</filter-class>
         <init-param>
           <param-name>casServerUrlPrefix</param-name>
-          <param-value>${environment.build.cas.protocol}://${environment.build.cas.server}${environment.build.cas.context}</param-value>
+          <param-value>http://localhost:8080/cas</param-value>
```

Comment:
Why are we hardcoding localhost here? If this can be controlled via uPortal-start that is okay, but if not this is really bad. We don't run CAS on the same machine as uPortal (it's also a bad idea to do that).